sstables/sstables.cc

```diff
@@ -1440,8 +1440,8 @@ void sstable::maybe_rebuild_filter_from_index(uint64_t num_partitions) {
     // Replace the existing filter with a new one that can optimally represent the given num_partitions.
     // The rebuilding is done in-place as this method is only called before the sstable is sealed.
     _components->filter = utils::i_filter::get_filter(num_partitions, _schema->bloom_filter_fp_chance(), get_filter_format(_version));
-    sstlog.info("Rebuilding bloom filter {}: resizing bitset from {} bytes to {} bytes.", filename(component_type::Filter), curr_bitset_size,
-                static_cast<utils::filter::bloom_filter*>(_components->filter.get())->bits().memory_size());
+    sstlog.info("Rebuilding bloom filter {}: resizing bitset from {} bytes to {} bytes. sstable origin : {}", filename(component_type::Filter), curr_bitset_size,
+                static_cast<utils::filter::bloom_filter*>(_components->filter.get())->bits().memory_size(), origin);
```

Comment:
nit: extra space between origin and colon

Reply:
V2 removes the extra space.